ray-operator/test/support/test.go

```diff
@@ -121,3 +126,34 @@ func (t *T) NewTestNamespace(options ...Option[*corev1.Namespace]) *corev1.Names
 	})
 	return namespace
 }
+
+func (t *T) StreamKubeRayOperatorLogs() {
+	ctx, cancel := context.WithCancel(context.Background())
+	t.T().Cleanup(cancel)
+	pods, err := t.Client().Core().CoreV1().Pods("").List(ctx, metav1.ListOptions{
```

Comment:
What does `""` mean? Does it signify using the default namespace, or does it mean listing Pods in all namespaces?

Reply:
It means listing Pods in all namespaces.

The operator is not always deployed in the "ray-system" namespace. If a user follows the helm command in the `DEVELOPMENT.md` guide, the operator will be installed into the default namespace.

Therefore I choose to list from all namespace here.

Reply:
Maybe we can add a comment for this?

Reply:
Sure, added.